gui/mozregui/wizard.py

```diff
@@ -198,6 +215,20 @@ def options(self):
                 elif isinstance(value, QDate):
                     value = value.toPyDate()
                 options[fieldname] = value
+        if options['bisect_type'] == 'nightlies':
```

Comment:
Hmm looking at this code, it seems a bit redundant. Maybe you could refactorise this a bit, for example:
kind = "date" if options['bisect_type'] == 'nightlies' else "changeset"
if not options['find_fix']:
    options['good_' + kind ] = options.pop('start_' + kind)
    options['bad_' + kind] = options.pop('end_' + kind)
else:
    options['good_' + kind] = options.pop('end_' + kind)
    options['bad_' + kind] = options.pop('start_' + kind)

This way we don't duplicate the logic.